src/main/java/org/example/C05_P50_Spiral_Matrix.java

```diff
@@ -0,0 +1,52 @@
+package org.example;
+/******************************************************************************
+ 50.Write a program in java to print a matrix in spiral form.
+ Expected Output :
+ The given array in matrix form is :
+ 1  2  3  4  5
+ 6  7  8  9  10
+ 11 12 13 14 15
+ 16 17 18 19 20
+ The spiral form of above matrix is:
+ 1 2 3 4 5 10 15 20 19 18 17 16 11 6 7 8 9 14 13 12
+ *******************************************************************************/
+public class C05_P50_Spiral_Matrix {
+    private C05_P50_Spiral_Matrix () {}
+    public static String makeArraySpiral(int[][] matrix){
+        int n=4;
```

Comment:
Don't use hardcoded values. n should be dynamic in order for your solution to work for all matrix sizes.